my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/mapper/ContactInformationMapper.java

```diff
@@ -145,7 +146,7 @@ private static JSONObject jsonifyEmergencyContact(ContactInformation eci) {
     JSONObject emergencyContact = new JSONObject();
     emergencyContact.put("EMERGENCY NAME", eci.getPreferredName());
     emergencyContact.put("RELATION", eci.getRelationship());
-    emergencyContact.put("RELATION COMMENT", eci.getComments());
+    emergencyContact.put("RELATION COMMENT", eci.getComments()+" ");
```

Comment:
I had this thought too.  I was on the fence and could go either way.  Spaces vs computational logic.  Let's do it.  I'll update.